rs/cycles_account_manager/tests/cycles_account_manager.rs

```diff
@@ -1,12 +1,12 @@
 use ic_base_types::NumSeconds;
 use ic_config::subnet_config::CyclesAccountManagerConfig;
-use ic_cycles_account_manager::WasmExecutionMode;
 use ic_cycles_account_manager::{IngressInductionCost, ResourceSaturation};
 use ic_interfaces::execution_environment::CanisterOutOfCyclesError;
 use ic_limits::SMALL_APP_SUBNET_MAX_SIZE;
 use ic_logger::replica_logger::no_op_logger;
 use ic_management_canister_types::{CanisterIdRecord, Payload, IC_00};
 use ic_registry_subnet_type::SubnetType;
+use ic_replicated_state::canister_state::execution_state::WasmExecutionMode;
 use ic_replicated_state::{
     canister_state::system_state::CyclesUseCase, testing::SystemStateTesting, SystemState,
 };
```

Comment:
Merge the `use`s? I think the same applies to most of the other locations where the `use` is added.

Reply:
done, also in other places, thanks!